{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception", "trace": "def test_broken():\n>       raise Exception()\nE       Exception\n\ntest_alluredemo.py:20: Exception"}, "start": 1622511114103, "stop": 1622511114104, "uuid": "36c0e2b0-d90e-4e01-bea3-148e84385e39", "historyId": "fb799eecaee64eb20ac70a0047ddd7ff", "testCaseId": "54327feac6bee8ccd45a88d670bf2886", "fullName": "test_alluredemo#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo"}, {"name": "host", "value": "WJYF2003-0261"}, {"name": "thread", "value": "24456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo"}]}